settings.gradle.kts

```
@@ -41,7 +41,8 @@ dependencyResolutionManagement {
     }
     // r2dbc is not compatible with earlier versions
     addSpringBootCatalog("springBoot2", "2.6.15", "2.+")
-    addSpringBootCatalog("springBoot3", "3.0.0", "3.+")
+    // spring boot 3.0 is not compatible with graalvm native image
+    addSpringBootCatalog("springBoot3", "3.1.0", "3.+")
```

does this make more sense?
```suggestion
    addSpringBootCatalog("springBoot31", "3.1.0", "3.1.+")
```

---

the latest version should still be 3.3

---

isn't testing of the latest version covered below by

```
    addSpringBootCatalog("springBoot32", "3.2.0", "3.+")
```

---

no, `springBoot32` is used by the module that has 3.2 as min version